examples/error-handling-in-single-place/error_handling_in_single_place.out

```diff
@@ -4,5 +4,3 @@ $ ballerina run error_handling_in_single_place.bal
 Error caught: InvalidAccountID, Account ID: -1
 Error caught: AccountNotFound, Account ID: 200
 Error caught during parsing: {ballerina/lang.int}NumberParsingError
-[ballerina/http] started HTTP/WS listener 192.168.1.18:34525
```

Comment:
These lines will be printed again right?

Reply:
No. This is the output of the on fail BBE. It doesn't use JDBC.